docs/07-faq/020-supported-clouds.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Which clouds are supported by Wing?
+id: supported-clouds
+keywords: [faq, supported clouds, winglang, Wing programming language, Wing language, AWS, GCP, Azure]
+---
+
+Wing was built from the ground up to make it an ideal choice for building applications on any cloud.
+Since wing is in early stages of developemnt it only supports the three biggest public clouds, with different levels of maturity:
```

Comment:
```suggestion
Winglang was built from the ground up to make it an ideal choice for building applications on any cloud. Every app has access to the Wing SDK, a set of batteries-included resources that represent cloud services that are common to most major cloud providers for common use cases.

Since Wing is in early stages of development, the Wing SDK supports each cloud with different levels of maturity:
```